java/core/src/test/org/apache/orc/TestOrcTimezone1.java

```diff
@@ -69,11 +70,11 @@ private static Stream<Arguments> data() {
         Arguments.of("Asia/Saigon", "Pacific/Enderbury"),
         Arguments.of("UTC", "Asia/Jerusalem")
 
-        // NOTE:
-        // "1995-01-01 03:00:00.688888888" this is not a valid time in Pacific/Enderbury timezone.
-        // On 1995-01-01 00:00:00 GMT offset moved from -11:00 hr to +13:00 which makes all values
-        // on 1995-01-01 invalid. Try this with joda time
-        // new MutableDateTime("1995-01-01", DateTimeZone.forTimeZone(readerTimeZone));
+    // NOTE:
+    // "1995-01-01 03:00:00.688888888" this is not a valid time in Pacific/Enderbury timezone.
+    // On 1995-01-01 00:00:00 GMT offset moved from -11:00 hr to +13:00 which makes all values
+    // on 1995-01-01 invalid. Try this with joda time
+    // new MutableDateTime("1995-01-01", DateTimeZone.forTimeZone(readerTimeZone));
```

Comment:
This new change looks wrong to me. Please recheck this change.

Reply:
It was crying for indentation to be brought at that level. It did felt weird.
Changed it to multiline comments

Reply:
In that case, let's add a suppression line for `CommentsIndentation` rule for that specific file because the original code might be more user-friendly.

Reply:
Got it, I would like to do the similar suppressions at 2 other places for readability.